examples/handlers/handlers/app.py

```diff
@@ -25,17 +24,6 @@ def do_function(self, widget, **kwargs):
             random.randint(0, 100)
         )
 
-    def do_generator(self, widget, **kwargs):
-        """A generator-based handler."""
-        # The generator yields a number; that number is the number of seconds
```

Comment:
There's also a generator handler in toga_demo.

Reply:
Good catch!